### PR TITLE
Added widget & text alignment

### PR DESCRIPTION
**Pull request by ppizarror**
This is a major change on element positioning, both textmenu and menu changes the way elements are aligned

### CODE REVIEW

**Comment by ppizarror** [comment]
@anxuae This will be part of v2.0, have added "align" into each widget

**Comment by anxuae** [comment]
Look very nice, good job!

File: pygameMenu/menu.py [hunk 1]
```diff
@@ -260,18 +259,24 @@ def __init__(self,
         # Init mouse
         self._mouse = mouse_enabled
 
-    def add_option(self, element_name, element, *args):
+    def add_option(self, element_name, element, *args, align=''):
```
Comment (anxuae) [inline pygameMenu/menu.py]: This is a syntax error in python2, maybe indicate in readme that only Python3 is supported?

Comment (ppizarror) [reply pygameMenu/menu.py]: @anxuae humm, by this only error I don't feel necessary to break compatibility. What else could i do? Maybe using **kwargs..

**Comment by ppizarror** [comment]
Fixed python2 compatibility

File: pygameMenu/menu.py (hunk 1 above)
Comment (ppizarror) [reply pygameMenu/menu.py]: kwargs works really good, by this weekend I'll work on README and minor stuff to publish v2.0

**Pull request merged**